src/react/src/components/CookiePreferencesText.jsx

```diff
@@ -3,25 +3,16 @@ import { InfoLink, InfoPaths } from '../util/constants';
 
 const CookiePreferencesText = () => (
     <div>
-        The Open Supply Hub uses cookies to collect and analyze site performance
-        and usage. By clicking the Accept button, you agree to allow us to place
-        cookies and share information with Google Analytics. For more
-        information, please visit our{' '}
+        We use cookies to give you the best experience on Open Supply Hub and
+        understand site performance. By accepting, you consent to our use of
+        cookies and other analytics tools according to our{' '}
         <a
             href={`${InfoLink}/${InfoPaths.termsOfService}`}
             target="_blank"
             rel="noreferrer"
         >
-            Terms of Service
+            privacy policy
         </a>{' '}
```

Comment:
Why keep `{' '}` here?

Reply:
+1

Reply:
It's just the space before the dot.
<img width="113" alt="Screenshot 2024-05-20 at 10 52 24" src="https://github.com/opensupplyhub/open-supply-hub/assets/9516550/b7f00033-e176-4eb1-9d78-78072b9c0b9b">